spec/forms/hub/create_client_form_spec.rb

```diff
@@ -108,6 +108,34 @@
         expect(tax_returns.map(&:service_type).uniq).to eq ["drop_off"]
       end
 
+      context "mixpanel" do
+        let(:fake_tracker) { double('mixpanel tracker') }
+        let(:fake_mixpanel_data) { {} }
+
+        before do
+          allow(MixpanelService).to receive(:data_from).and_return(fake_mixpanel_data)
+          allow(MixpanelService).to receive(:send_event)
+        end
+
+        it "sets intake.visitor_id" do
```

Comment:
Now that `visitor_id` is required on`Intake`, I think you can remove this test, given that `it "creates an intake"` passes.